### PR TITLE
Add new character

### PR DESCRIPTION
**Pull request by Ploypiti**
### My steps to solve it,
Changes: Add `pixie.svg` and edit `gui.js`
[>>Deploy Links<<](http://rawgit.com/Ploypiti/labyrinth/master/index.html)

P.S. The character looks _extremely_ small in the Labyrinth and that's kinda depressing 😭 Nothing can be seen at all and every single detail has gone waste...Also, the image isn't cracked--I was experimenting with some textures on illustrator. Hope you like it ❤️ 

pixie

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
@Ploypiti it renders but takes a lot of time, atleat that what happened in my screen

**Comment by arjunpandey2003** [comment]
screen shot 2018-01-07 at 9 53 47 am

**Comment by Ploypiti** [comment]
Ping @niccokunzmann.

**Comment by niccokunzmann** [comment]
@Ploypiti Thanks for this wonderful character. There are merge conflicts.

I have two ideas:
- faster loading: create a smaller version of the character with `_small` in the file name with the dimensions from the documentation in the README file.
- You can create a new tile and use linking instead of embedding of the character to include your original character. E.g. this character can be in a library and you can get a book which you can read in the chair and then get a magic spell.

**Comment by abishekvashok** [comment]
> You can create a new tile and use linking instead of embedding of the character to include your original character. E.g. this character can be in a library and you can get a book which you can read in the chair and then get a magic spell.

Wow, you seem to be poetic and a short story writer. That will be great!

File: js/gui.js [hunk 1]
```diff
@@ -5,6 +5,25 @@
...
+            "characters/robo.svg": "Robo",
+            "characters/swammy.svg": "Swammy",
+            "characters/monster.svg": "Monster",
+            "characters/pixie.svg": "Forest Spirit",
+            "characters/elephant.svg": "Elephant",
+            "characters/shadowman.svg": "Shadowman",
+            "characters/santa.svg": "Santa Clause",
+            "characters/detective.svg": "Agent X"
+        });
+
```
Comment (niccokunzmann) [inline js/gui.js]: Please have a look at the lines above, they are duplicated.

File: js/gui.js [hunk 2]
```diff
@@ -23,6 +42,7 @@ window.addEventListener("load", function()
       "characters/santa.svg": "Santa Clause",
       "characters/detective.svg":"Agent X",
       "characters/Geekomoji.svg": "Geekomoji",
+
```
Comment (niccokunzmann) [inline js/gui.js]: your character should go here.

**Comment by Ploypiti** [comment]
@niccokunzmann I'd resizes the character for now, I'd love to add my character as a part of a tile, though it could be a Doppelgänger if the character meets another piece of itself. So, basically the idea of the spellbook thing is pretty cool, imma create library tiles with a different character, later. Thanks for your suggestion ❤️

**Comment by niccokunzmann** [comment]
@Ploypiti One idea to resolve the conflicts is this:
1. create a new branch based on the master branch. `git checkout master; git checkout -b add-character-3`
2. do `git cherry-pick 78fd1e0 ` to get this one commit where you added the character
3. edit the gui.js fie again
4. push and create a new pull request.

If you want to take 15 minutes for learning how to use the command line, you can have a look at http://try.github.io 
If I can help you with anything, please let me know. Can I answer a question?

**Comment by Ploypiti** [comment]
@niccokunzmann Can I close this PR?

**Pull request closed without merging**